mixins/_visuallyhidden.scss

```diff
@@ -0,0 +1,16 @@
+//
+// Visually Hidden
+// --------------------
+//
+//
+
+@mixin visuallyhidden {
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  clip: rect(0 0 0 0);
+}
```

Comment:
1. Why not use `a11y`for this which does exactly what we want? If someone who uses this in the past could get irritated if it no longer exists. I pushed an update sou you can have a look.
2. Please use a hyphen if you use two words e.g. `visually-hidden` instead of `visually-hidden`.